python/tvm/auto_scheduler/utils.py

```diff
@@ -30,7 +30,7 @@
 try:
     import psutil
 except ImportError:
-    raise ImportError("psutil not found, try `pip install psutil` to fix this")
```

Comment:
I prefer keeping this ImportError. During internal development, we have met several times our environment doesn't import this package, we could tune but meet process error and find this root cause is hard.